tools/windows-build.bat

```diff
@@ -20,7 +20,7 @@ dir wheelhouse
 rem cibuildwheel installs the generated packages, but they're not ready yet,
 rem so remove them.
 rem FIXME: this only covers python27 (default)
-pip uninstall -y confluent_kafka
+pip uninstall -y confluent_kafka[dev]
```

Comment:
what's dev?

Reply:
everything, so pytest, avro, requests etc

Reply:
Is that a standard package thing, or a sub-package that we specifically make?

Reply:
its an "extra", you can see it in setup.py